### PR TITLE
Code of Conduct—we need one.

### PR DESCRIPTION
**Pull request by strand**

### CODE REVIEW

File: code-of-conduct/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+We invite all those who participate in Open Source and Feelings to help us create safe and positive experiences for everyone.
+
+#### Open Citizenship
+
+A supplemental goal of this Code of Conduct is to increase open citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+
+Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+
+If you see someone who is making an extra effort to ensure our community is safe, welcoming, inclusive friendly, and encourages all participants to contribute to the fullest extent, we want to know.
```
Comment (kerrizor) [inline code-of-conduct/code-of-conduct.md]: I almost want to see this line in **bold**

File: code-of-conduct/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+#### Code of Conduct
+
+All attendees, speakers, sponsors and volunteers at Open Source & Feelings are required to abide by the code of conduct. Organisers will enforce this code throughout the event. We expect cooperation from all participants to help ensure a safe environment for everyone.
```
Comment (kerrizor) [inline code-of-conduct/code-of-conduct.md]: missing a serial comma

Also, add "organizers" to the list

File: code-of-conduct/code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+ - Refrain from demeaning, discriminatory, or harassing behavior and speech.
+ - Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+ - Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
+
+##### Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+ - Violence, threats of violence or violent language directed against another person.
+ - Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
```
Comment (kerrizor) [inline code-of-conduct/code-of-conduct.md]: serial comma

File: code-of-conduct/code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+ - Posting or displaying sexually explicit or violent material.
+ - Posting or threatening to post other people’s personally identifying information ("doxing").
+ - Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+ - Inappropriate photography or recording.
+ - Inappropriate physical contact. You should have someone’s consent before touching them.
+ - Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
+ - Deliberate intimidation, stalking or following (online or in person).
+ - Advocating for, or encouraging, any of the above behavior.
+ - Sustained disruption of community events, including talks and presentations.
```
Comment (kerrizor) [inline code-of-conduct/code-of-conduct.md]: do these lines need periods at the end?

File: code-of-conduct/code-of-conduct.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+The following behaviors are considered harassment and are unacceptable within our community:
+
+ - Violence, threats of violence or violent language directed against another person.
+ - Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
+ - Posting or displaying sexually explicit or violent material.
+ - Posting or threatening to post other people’s personally identifying information ("doxing").
+ - Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+ - Inappropriate photography or recording.
+ - Inappropriate physical contact. You should have someone’s consent before touching them.
+ - Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
```
Comment (kerrizor) [inline code-of-conduct/code-of-conduct.md]: the comma after includes is weird, as is the semicolon after jokes

File: code-of-conduct/code-of-conduct.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+ - Deliberate intimidation, stalking or following (online or in person).
```
Comment (kerrizor) [inline code-of-conduct/code-of-conduct.md]: image

**Comment by kerrizor** [comment]
Thanks for doing this, yo! :+1:

File: code-of-conduct/code-of-conduct.md (hunk 6 above)
Comment (strand) [inline code-of-conduct/code-of-conduct.md]: I <3 you.

File: code-of-conduct/enforcement.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+A participant may be expelled by the decision of organizers for whatever reasons they deem sufficient. However, here are some general guidelines for when a participant should be expelled:
+
+ - A single serious offense (e.g., punching or groping someone)
+ - A single obviously intentional offense (e.g., taking photos without permission)
+ - A second offense resulting in a warning from staff
+ - Continuing to harass after any "No" or "Stop" instruction
+ - A pattern of harassing behavior, with or without warnings
+
+Security and local authorities should be contacted when appropriate.
```
Comment (khjrtbrg) [inline code-of-conduct/enforcement.md]: Can we contact authorities on other's behalf? If so, that might be good to add.

Comment (strand) [inline code-of-conduct/enforcement.md]: I believe so. What should that language look like?

File: code-of-conduct/long.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+Open Source & Feelings is dedicated to providing a safe, inclusive, welcoming, harassment-free conference experience for everyone regardless of gender, gender identity, gender expression, sexual orientation, ability, physical appearance, body size, socioeconomic status, race, age, or religion (or lack thereof).
+
+### Expected Behavior
+
+The following behaviors are expected and requested of all community members:
+
+ - Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+ - Exercise consideration and respect in your speech and actions.
+ - Attempt collaboration before conflict.
+ - Refrain from demeaning, discriminatory, or harassing behavior and speech.
```
Comment (roseaboveit) [inline code-of-conduct/long.md]: This is the only bullet point in this section that refers to what participants should not do. Perhaps we should try to flip it around to a positive, or just make sure it is all covered in the unacceptable behavior.

File: code-of-conduct/long.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+ - Violence, threats of violence or violent language directed against another person
+ - Sexist, racist, homophobic, transphobic, ableist, or otherwise discriminatory jokes and language
+ - Posting or displaying sexually explicit or violent material
+ - Posting or threatening to post other people’s personally identifying information ("doxing")
+ - Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability
+ - Inappropriate photography or recording
+ - Inappropriate physical contact. You should have someone’s consent before touching them
```
Comment (roseaboveit) [inline code-of-conduct/long.md]: I'd like to consider making this statement stronger. Something like:

You should have someone’s explicit consent before touching them in any way

File: code-of-conduct/long.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+ - Unwelcome sexual attention. This includes sexualized comments or jokes, inappropriate touching, groping, and unwelcome sexual advances
+ - Deliberate intimidation, stalking, or following (online or in person)
+ - Advocating for, or encouraging, any of the above behavior
+ - Sustained disruption of community events, including talks and presentations
+
+#H4 Harassment
+
+We do not tolerate harassment of participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks. Participants violating these rules may be sanctioned or expelled from the conference without a refund at the discretion of the conference organizers.
+
+Sponsors are subject to the Code of Conduct. In particular, sponsors should not use sexualized images, activities, or other material. Booth staff, including volunteers, should not wear sexualized clothing, or otherwise create a sexualized environment.
```
Comment (roseaboveit) [inline code-of-conduct/long.md]: Is there some way we are going to define the distinction between sexualized clothing and gender expression? Do we in essence have a dress code? And if so, should we make it explicit?

File: code-of-conduct/long.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+Harassment includes:
+
+ - verbal comments that reinforce social structures of domination related to gender, gender identity, gender expression, sexual orientation, disability, physical appearance, body size, race, age, and religion (or lack thereof)
+ - sexual images in public spaces
+ - deliberate intimidation
```
Comment (roseaboveit) [inline code-of-conduct/long.md]: I think if we are going to specify "deliberate" intimidation, we should also add "deliberate" to the first point. Given the entrenchment of these social structures of domination in this society, I think that unintentional mistakes that can be turned into learning opportunities are even more likely with point 1 circumstances than with intimidation.

File: code-of-conduct/long.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+ - [TKTKTK Phone number for conference security or organizers]
+ - [TKTKTK Phone number for hotel/venue security]
+ - [TKTKTK Local law enforcement]
+ - [TKTKTK Local sexual assault hot line]
+ - [TKTKTK Local emergency and non-emergency medical (e.g., urgent care, day clinic)]
+ - [TKTKTK Local taxi company]
+
+### Scope
+
+We expect all community participants (guests, sponsors, and contributors, paid or otherwise) to abide by this Code of Conduct in all community venues–online and in-person—as well as in all one-on-one communications pertaining to community business.
```
Comment (roseaboveit) [inline code-of-conduct/long.md]: These dashes are confusing. Maybe parentheses? Is that grammatically okay? (online and in-person)

File: code-of-conduct/long.md (hunk 8 above)
Comment (strand) [inline code-of-conduct/long.md]: I think either pull into unacceptable behavior or use emphasis to indicate that it is different.

File: code-of-conduct/long.md (hunk 10 above)
Comment (strand) [inline code-of-conduct/long.md]: I have no idea how we can actually define "sexualized clothing." :-/

**Pull request merged**